src/libcore/fmt/mod.rs

```diff
@@ -776,6 +776,30 @@ pub trait UpperExp {
 ///
 ///   * output - the buffer to write output to
 ///   * args - the precompiled arguments generated by `format_args!`
+///
+/// # Examples
+///
+/// Basic usage:
+///
+/// ```
+/// use std::fmt;
+///
+/// let mut output = String::new();
+/// fmt::write(&mut output, format_args!("Hello {}!", "world")).unwrap();
```

Comment:
Yes, but `unwrap()` is not the idiomatic way to handle errors, and we don't want to show unidiomatic code.